contentgrid-spring-swagger/src/main/resources/swagger-initializer.js

```diff
@@ -0,0 +1,20 @@
+window.onload = function() {
+    //<editor-fold desc="Changeable Configuration Block">
+
+    // the following lines will be replaced by docker/configurator, when it runs in a docker-container
```

Comment:
This is not exactly true; that's a comment coming from where the file was copied from, bus is not accurate in this project

Reply:
I copied it varbatim from the current project. I'll clean it up.